content/influxdb/v2.0/telegraf-configs/update.md

```diff
@@ -40,7 +37,38 @@ To modify existing Telegraf configuration settings,
    to update the name or description.
 3. Press **Return** or click out of the editable field to save your changes.
 
+### Edit the configuration file directly in the UI
+
+1. In the navigation menu on the left, select **Data** (**Load Data**) > **Telegraf**.
+
+    {{< nav-icon "load data" >}}
+
+2. To edit the configuration file:
+  - Click the name of the configuration.
+  - Add or update [Telegraf plugin settings](/telegraf/latest/plugins/) in the window that appears.
+  {{% note %}}
+  The text editor window doesn't detect if any plugins or settings are misconfigured. Any errors in your configuration that may cause Telegraf to fail will be displayed when you run Telegraf in the CLI.
+  {{% /note %}}
+  - Click **Save Changes** and then **Save** again to confirm.
+3. In order for your changes to take affect, you must re-run your updated Telegraf configuration using the `-config` flag. To find the exact command for your configuration file, click **Setup Instructions** from the **Telegraf** page.
```

Comment:
In order for your changes to take affect, you must re-start each Telegraf agent that you wish to use the modified configuration using the `-config` flag.

Reply:
see my comment on line 50. 

Also, i think it's `take effect` not `affect`